sceptre/template_handlers/http.py

```diff
@@ -75,9 +77,28 @@ def _get_template(self, url):
         :rtype: str
         """
         self.logger.debug("Downloading file from: %s", url)
+        session = self._get_retry_session()
         try:
-            response = requests.get(url)
+            response = session.get(url, timeout=(2, 5))
             return response.content
-        except requests.exceptions.RequestException as e:
+        except Exception as e:
```

Comment:
Is there a more specific exception we are expecting here? I realize that changing request to session might have changed the exception here.

Reply:
fixed